### PR TITLE
Modify existing pyre-check action to use newly added native sarif support

### PR DESCRIPTION
**Pull request by abishekvashok**
Modifies existing pyre-check github action to use the newly added sarif
output support. Also remove redundant scripts.

Signed-off-by: Abishek V Ashok <abishekvashok@gmail.com>
Aftermath of a new pyre-check version being released on pypi with sarif support!

Test plan:
See the GitHub action!

### CODE REVIEW

File: .github/workflows/pyre.yml [hunk 1]
```diff
@@ -14,31 +14,20 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-python@v1
+      - uses: actions/setup-python@v2
 
-      - name: Install Dependencies
+      - name: Install Dependencies and pyre-check
```
Comment (dkgi) [inline .github/workflows/pyre.yml:19]: Why the change here? I think of this as "dependencies for the action" and Pyre is just a normal dependency.

Comment (abishekvashok) [reply .github/workflows/pyre.yml]: I wanted to give it some special status but yeah it's fine either way :)

**Comment by abishekvashok** [comment]
@dkgi thanks for the review, removed the unnecessary change :)
